timemachine/cpp/src/constants.hpp

```diff
@@ -0,0 +1,3 @@
+
+static const double BOLTZ = 0.008314462618;  // kJ/mol per kelvin, molar gas constant / 1000
```

Comment:
tiny nit: I find the `/ 1000` slightly confusing since the units of the molar gas constant aren't specified. Maybe just
```suggestion
static const double BOLTZ = 0.008314462618;  // molar gas constant in kJ/mol per kelvin
```
?